tests/unit/plugins/janitor/test_reconciler.py

```diff
@@ -180,13 +180,8 @@ async def validate_rrsets_by_zone():
     await recon_client.cleanup()
 
     assert exp_log_records == len(caplog.records)
-    if exp_log_records == 2:
-        # it's in a cancelling state which can't be directly tested
-        assert not coro1.done()
-        assert not coro2.done()
-    else:
-        assert coro1.done()
-        assert coro2.done()
```

Comment:
I don't totally understand what this test was trying to test either, but if you do, +1

Reply:
No, not really.  Just the behavior apparently changed wrt something about cancelling requests.